docs/source/pickling_cuml_models.ipynb

```diff
@@ -183,6 +183,70 @@
    "source": [
     "single_gpu_model.cluster_centers_"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exporting cuML Random Forest models for inferencing on machines without GPUs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Starting with cuML version 21.06, you can export cuML Random Forest models and run predictions with them on machines without an NVIDIA GPUs. The [Treelite](https://github.com/dmlc/treelite) package defines an efficient exchange format that lets you portably move the cuML Random Forest models to other machines. We will refer to the exchange format as \"checkpoints.\"\n",
```

Comment:
I think adding a link to here from the main docstring of the RF classes (with a note like `RF classes can be exported to Treelite for inference on machines without GPUs` or so) would be useful for people looking at the api docs

Reply:
Done